subsys/testsuite/ztest/Kconfig

```diff
@@ -87,3 +87,29 @@ config ZTEST_PARAMETER_COUNT
 	depends on ZTEST_MOCKING
 	help
 	  Maximum amount of concurrent return values / expected parameters.
+
+config ZTRESS
+	bool "Stress test framework"
+	select THREAD_RUNTIME_STATS
+	select THREAD_MONITOR
+	select TEST_RANDOM_GENERATOR
+	depends on !USERSPACE
```

Comment:
Hi @nordic-krch , here is my suggestion:
```
--- a/subsys/testsuite/ztest/Kconfig
+++ b/subsys/testsuite/ztest/Kconfig
@@ -92,8 +92,7 @@ config ZTRESS
        bool "Stress test framework"
        select THREAD_RUNTIME_STATS
        select THREAD_MONITOR
-       select TEST_RANDOM_GENERATOR
-       depends on !USERSPACE
+       select TEST_RANDOM_GENERATOR if !ENTROPY_HAS_DRIVER
```

Reply:
thanks for suggestion. CI is green